apps/app-shell-app/src/app/app.component.ts

```diff
@@ -78,7 +78,7 @@ export class AppComponent implements OnInit {
         this.appShell.shellBar.setTitle('One Procurement');
         this.appShell.shellBar.setSubTitle('Solution for all');
 
-        this.appShell.messageBus.sendTo('error:event',
+        this.appShell.messageBus.send('error:event',
```

Comment:
There is app-shell-app under `apps` when you changed signature please update the app as well. I should not compile if you do ng build app-shell-app